ontology/iceo.md

```diff
@@ -16,21 +16,14 @@ license:
 preferredPrefix: ICEO
 products:
 - id: iceo.owl
+publications:
+- id: https://www.ncbi.nlm.nih.gov/pubmed/35058462
+  title: ICEO, a biological ontology for representing and analyzing bacterial integrative and conjugative elements
 repository: https://github.com/ontoice/ICEO
 tracker: https://github.com/ontoice/ICEO/issues
 activity_status: active
 ---
 
-# ICEO: Integrative and Conjugative Element Ontology
-
 ICEO is a community-driven ontology to represent, standardize, and integrate bacterial integrative and conjugative elements (ICEs) and to support computer-assisted reasoning.
 
 The integrative and conjugative elements (ICEs) are modular mobile genetic elements that are integrated into a host genome and are passively propagated during chromosomal replication and cell division. We generated an ICE database called ICEberg (http://db-mml.sjtu.edu.cn/ICEberg/). The ICEO is developed to first systematically represent the ICE information collected in the database. The ICEO will also be further used to enhance the ICEberg and support various applications.
```

Comment:
@Lemon-Liu interestingly, I don't see ICEO used at all in ICEberg. Is this an oversight?

Reply:
Hi @cthoyt, I am sorry for the late response and I appreciate your question. ICEO was developed to ontologically represent and integrate the ICE gene information and functional modules to support automatically computer-assisted reasoning. And ICEO and ICEO-based features are being explored to be integrated into ICEberg. For example,
- We recently developed an [ICEberg integrated SPARQL query interface](https://bioinfo-mml.sjtu.edu.cn/ICEberg2/search_sparql.php) with some useful query models. Users can implement some commonly used SPARQL queries toward ICEO by selecting or inputting keywords without the need of learning and writing SPARQL query scripts. And more query models will be added under the users’ requests. 
- We linked each ICEberg item and the ICEO contents to make a seamless interaction between these two resources. Using [ICEKp1 as an example](https://bioinfo-mml.sjtu.edu.cn/ICEberg2/feature_page.php?ice_id=126), a click of the ICEO ID on the ICEberg page allows a user to access the Ontobee website and identify more detailed information about the ICE. 

And the relevant work has been published recently: 
 - Meng Liu#, Jialin Liu#, Guitian Liu, Hui Wang, Xiaoli Wang, Zixin Deng, Yongqun He, Hong-Yu Ou. ICEO, a biological ontology for representing and analyzing the bacterial integrative and conjugative element. Scientific Data, 2022, 9(1):11.

Reply:
And the relevant link is https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8776819/. I just found out that I may miss the proper information about adding the publication, could you please help add this one for ICEO?

Reply:
That's great, thanks. This corresponds to the publication I already found and added in this PR!

Reply:
Glad to see ICEO has been recently integrated in ICEberg, too :) what a cool resource and such interesting biology I got to learn about from your work